frontend/ui/widget/inputdialog.lua

```diff
@@ -197,7 +197,7 @@ function InputDialog:init()
         }
     }
     --little hack to piggyback on the layout of the button_table to handle the new InputText
-    table.insert(self.button_table.layout, #self.button_table.layout, {self._input_widget})
+    table.insert(self.button_table.layout, 1, {self._input_widget})
```

Comment:
Could you explain what this little hack is doing exactly? Seeing a specific number like `1` is a lot more worrisome than #self.button_table.layout. ;-)